docs/source/reference/config.rst

```diff
@@ -40,6 +40,18 @@ Available fields and semantics:
     - gcp
     - kubernetes
 
+  nvidia_gpu:
+    # Disable ECC for NVIDIA GPUs (optional).
+    #
+    # Set to true to disable ECC for NVIDIA GPUs during provisioning. This is
+    # useful to improve the GPU performance in some cases (up to 30%
+    # improvement). This will only be applied if a cluster is requested with
+    # NVIDIA GPUs.
+    # https://portal.nutanix.com/page/documents/kbs/details?targetId=kA00e000000LKjOCAW.
```

Comment:
Link seems to require login. Any other ref?

Reply:
It is the issue with the ending `.`. Just removed it.